### PR TITLE
tests: add some tests to run on Travis

### PR DESCRIPTION
**Pull request by giuseppe**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - libslang2-dev
+      - gettext
+      - zlib1g-dev
+      - libselinux1-dev
+      - debhelper
+      - lsb-release
+      - pkg-config
+      - po-debconf
+      - autopoint
+      - libtool
```
Comment (AkihiroSuda) [inline .travis.yml]: Can we dockerize (yes, you can set `alias docker=podman` :P) the test env and we can almost get rid of these extra deps?

File: tests/common.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        fi
+        let COUNTER=COUNTER+1
+    done
+}
+
+function wait_for_network_device {
+    # Wait that the device appears.
+    COUNTER=0
+    while [ $COUNTER -lt 10 ]; do
+        if nsenter --preserve-credentials -U -n --target=$1 ifconfig $2; then
```
Comment (AkihiroSuda) [inline tests/common.sh]: s/ifconfig/ip/g ?

File: tests/test-slirp4netns-configure.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -xeuo pipefail
+
+. $(dirname $0)/common.sh
+
+unshare -r -n sleep 1000 &
```
Comment (AkihiroSuda) [inline tests/test-slirp4netns-configure.sh]: `sleep infinity`?

File: tests/test-slirp4netns-configure.sh [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+child=$!
+
+wait_for_network_namespace $child
+
+slirp4netns -c $child tun11 &
```
Comment (AkihiroSuda) [inline tests/test-slirp4netns-configure.sh]: `11`?

File: tests/test-slirp4netns-configure.sh [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+slirp_pid=$!
+
+wait_for_network_device $child tun11
+
+function cleanup {
+    kill -9 $child $slirp_pid
+}
+trap cleanup EXIT
+
+nsenter --preserve-credentials -U -n --target=$child ifconfig -a | grep tun11
```
Comment (AkihiroSuda) [inline tests/test-slirp4netns-configure.sh]: Maybe we can use `ip -j` and `jq`?
`ip -j` might not be available in older distros, but not a severe problem if we containerize the test env.

File: tests/test-slirp4netns-configure.sh (hunk 4 above)
Comment (giuseppe) [reply tests/test-slirp4netns-configure.sh]: just a random number, so that is less likely that the device was configured by some other tool

Comment (AkihiroSuda) [reply tests/test-slirp4netns-configure.sh]: image

File: tests/common.sh (hunk 2 above)
Comment (giuseppe) [reply tests/common.sh]: done

File: tests/test-slirp4netns-configure.sh (hunk 3 above)
Comment (giuseppe) [reply tests/test-slirp4netns-configure.sh]: done

File: .travis.yml (hunk 1 above)
Comment (giuseppe) [reply .travis.yml]: is it something we could do in a different PR?  I realized I wanted a container as well when I needed to install updated software on Travis :-)

**Comment by AkihiroSuda** [comment]
CI failing

> See ./test-suite.log
> Please report to suda.akihiro@lab.ntt.co.jp

Can we change the string to "Please report to https://github.com/rootless-containers/slirp4netns/issues"?

**Pull request merged**